net-dns/maradns/maradns-3.5.0005.ebuild

```diff
@@ -17,12 +16,15 @@ SLOT="0"
 KEYWORDS="amd64 ~mips ~ppc x86"
 IUSE="examples ipv6"
 
-DEPEND="${PYTHON_DEPS}"
-RDEPEND="${DEPEND}
-	acct-user/maradns
+BDEPEND="${PYTHON_DEPS}"
+DEPEND="
 	acct-group/maradns
 	acct-user/duende
+	acct-user/maradns
 	dev-lang/perl"
+RDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}"/${P}-flags.patch )
 
 src_prepare() {
 	default
```

Comment:
usex or one of the others in src_configure? Maybe new line too

Reply:
Sure